gdx/src/com/badlogic/gdx/utils/ScreenUtils.java

```diff
@@ -70,13 +70,7 @@ public static TextureRegion getFrameBufferTexture (int x, int y, int w, int h) {
 	}
 
 	public static Pixmap getFrameBufferPixmap (int x, int y, int w, int h) {

```

Comment:
We could add a deprecation annotation here to remove this eventually and keep this non-breaking, or remove the method at all